### ISSUE TITLE
Enable adjustment of label anchor

### ISSUE THREAD

**Issue by dronecz** [gregdavill/KiBuzzard#103]
Hello, 

I searched issues, but did not find similar issue. 

When I try to make or edit text, my aligment does not work.
 
![kibuzzard_aligment](https://user-images.githubusercontent.com/16454728/225588156-3a5ce062-cd8e-4c79-b7d2-4726a21f8022.gif)

KiCAD: 7.0.1
Plugin version: 1.4.0
OS: Windows 10

**Comment by NPoole** [comment]
"Alignment" refers to the alignment of the text inside the rectangular portion of the label. I suspect it works correctly, but you will need to specify a larger value for "width" before you'll see the effect.

**Comment by dronecz** [comment]
@NPoole Yes, you are right. What I had in mind is this from the label generator in Eagle:
![image](https://github.com/gregdavill/KiBuzzard/assets/16454728/c832285b-bd12-4394-a5eb-9797f997b8b4)
I guess, as Kicad has just anchor in the middle of the footprint, this flag was not ported to KiBuzzard.

**Comment by UnexpectedMaker** [comment]
Yes please! I can't do centred or right aligned text within labels, which is kinda infuriating, as I do that kinda crazy all the time in my designs ;)

<img width="659" alt="Screenshot 2024-06-08 at 8 02 05 PM" src="https://github.com/gregdavill/KiBuzzard/assets/3156212/abe1b64e-1a31-4e52-8f98-a3b1a75fba31">
